smoke-test.go

```diff
@@ -0,0 +1,212 @@
+package main
+
+import (
+	"io"
+	"net/http"
+	"time"
+	"github.com/gophercloud/gophercloud/openstack"
+	"os"
+	"fmt"
+	"github.com/gophercloud/gophercloud"
+	"github.com/monasca/golang-monascaclient/monascaclient"
+	"strconv"
+	"github.com/monasca/golang-monascaclient/monascaclient/models"
+)
+
+var webhookTriggered bool = false
+var testsSucceeded int = 0
+var totalTests int = 6
```

Comment:
I wonder if it makes more sense to increment totalTests inside every test. That way adding a new test doesn't require updating this value